stories/line.stories.js

```diff
@@ -69,7 +69,7 @@ storiesOf('Line Chart', module)
   .add('Overlord adPosition', () => (
     <ResponsiveChartWrapper>
       <LineChart
-        title='Overlord adPosition'
+        title={ 'Overlord adPosition' }
```

Comment:
when no evaluation is needed but a simple string, prefer as-is.

we also need a more consistent and enforcing eslint ruleset for these, which I'll do later. feel free to leave this out